Sources/Client/Client/Client.swift

```diff
@@ -18,16 +18,62 @@ public final class Client: Uploader {
     public typealias OnEvent = (Event) -> Void
     
     /// A client config (see `Config`).
-    public static var config = Config(apiKey: "")
-    /// A shared client.
-    public static let shared = Client()
+    @available(*, deprecated, message: """
+    Configuring the shared Client using the static `Client.config` variable has been deprecated. Please create an instance
+    of the `Client.Config` struct and call `Client.configure(_:)` to set up the shared instance of Client.
+    """)
+    public static var config: Config = .init(apiKey: "_deprecated") {
```

Comment:
Why not empty api key?

Reply:
Because I didn't want to trigger an assertion failure just by someone doing `_ = Client.config`. Technically, there's nothing wrong with touching this variable. I used "_deprecated" as a note for future us that this is safe to remove.

Reply:
I didn't understand fully, can you clarify? How can `_ = Client.config` cause an assertion failure?

Reply:
Because we added `assert(!apiKey.isEmpty, "Empty string is not a valid apiKey.")` to `Config.init`.

Reply:
Ooooh I see, you're right! Makes sense 👍🏼